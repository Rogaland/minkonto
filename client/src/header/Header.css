.header {
    padding-top: 15px;
    padding-left: 15px;
}
.header img {
    max-width: 200px;
}