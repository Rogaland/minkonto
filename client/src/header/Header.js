import React, {Component} from 'react';

import logo from './rogfk_liggende_farger.jpg';
import './Header.css';

class Header extends Component {
    render() {
        return (
            <div className="header">
                <img src={logo} alt=''/>
            </div>
        )
    }
}

export default Header;