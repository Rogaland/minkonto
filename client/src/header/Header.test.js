import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {

  it('Renders Header component', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});