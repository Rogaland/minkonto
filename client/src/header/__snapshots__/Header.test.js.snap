// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Renders Header component 1`] = `
<div
  className="header"
>
  <img
    alt=""
    src="rogfk_liggende_farger.jpg"
  />
</div>
`;
